### DISCUSSION TITLE
How to customize the search paths of `add_requires`

### DISCUSSION THREAD

**Discussion by qiuwei** [xmake-io/xmake#2643] [General]
`add_requires`  searches paths such as `/usr/lib`, `/opt/lib` etc. How can I customize the paths it searches?

**Comment by waruqi** [comment]
you can use add_extsources or on_fetch to improve find packages in package()

https://github.com/xmake-io/xmake-repo/blob/52c27bf0940d3ffce261201c39911315854c3686/packages/g/glib/xmake.lua#L30

**Reply by qiuwei** [reply]
Sorry that I did not make it clear about what I am trying to do.
I installed boost in a customized location(/home/customize_lib). I tried using it in my project by `add_requries("boost_date_time", {system=true})` , how can I make xmake search `/home/customize_lib` to find boost?

**Reply by waruqi** [reply]
You need to configure PKG_CONFIG_PATH to ensure that pkg-config can find it.
